hooks/95-sbupdate.hook

```diff
@@ -10,6 +10,10 @@ Operation = Upgrade
 Operation = Remove
 Type = Path
 Target = usr/lib/initcpio/*
+Target = usr/lib/modules/*/vmlinuz
```

Comment:
Duplicate target entry. It had already added in [50-sbupdate-remove.hook](https://github.com/andreyv/sbupdate/blob/master/hooks/50-sbupdate-remove.hook#L4).
```suggestion
```